File: ios/Classes/PhonePhoto.swift
//
//  AlbumModel.swift
//  custom_image_picker
//
//  Created by Malas Taha on 12/29/19.
//

import Foundation

class PhonePhoto {
    let id:String
    let albumName:String
    let photoUri:String
    init(id:String, albumName:String, photoUri:String) {
      self.id = id
      self.albumName = albumName
      self.photoUri = photoUri
    }
}

File: android/src/main/kotlin/com/tahamalas/custom_image_picker/PhonePhoto.kt
package com.tahamalas.custom_image_picker

class PhonePhoto(val id: String, val albumName: String, val photoUri: String) {
    fun toJson() : String{
        return "{\"id\": \"$id\", \"albumName\": \"$albumName\", \"photoUri\": \"$photoUri\"}"
    }
}